src/shared/common/components/workstack.jsx

```diff
@@ -215,7 +215,9 @@ class WorkstackAllocate extends Component {
                                 onChange={handleChange.bind(this)}
                                 className={'govuk-checkboxes__input'}
                             />
-                            <label className='govuk-label govuk-checkboxes__label' htmlFor={`selected_cases_${item.caseUUID}`}></label>
+                            <label className='govuk-label govuk-checkboxes__label' htmlFor={`selected_cases_${item.caseUUID}`}>
+                                <span className='govuk-visually-hidden'>{item.caseReference}</span>
```

Comment:
Wouldn't a `aria-label` property achieve the same thing, without having to nest a span element?

Reply:
I'm not an expert here, but it looks like `aria-label` is there for when there's no associated label element, whereas here we have a label element that's blank?

I'm not sure _why_ we need a label element here, but without it the checkbox loses its big GOV.UK styling.